### PR TITLE
ZENKO-3255 add tar archive on documentation artifacts

### PR DESCRIPTION
**Pull request by tcarmet**
I've modified the location in which the documentation is stored in the artifacts.

It is now located under a folder `documentation` in which we can find:
* The html build
* The latex build
* The doctrees (sphinx stuff)
* A tar.gz file containing the 3 folder above

### CODE REVIEW

File: eve/main.yml [hunk 1]
```diff
@@ -225,22 +225,33 @@ stages:
...
-          tox --workdir /tmp/tox -e docs -- latexpdf BUILDDIR="%(prop:builddir)s/build/artifacts"
-          tox --workdir /tmp/tox -e docs -- latexpdf BUILDDIR="%(prop:builddir)s/build/artifacts"
+          tox --workdir /tmp/tox -e docs -- latexpdf BUILDDIR="%(prop:documentation_builddir)s"
+          tox --workdir /tmp/tox -e docs -- latexpdf BUILDDIR="%(prop:documentation_builddir)s"
         workdir: build/docs
-
+    - ShellCommand:
+        name: Create tar file for documentation
+        command: >-
+          tar --create -gzip
```
Comment (jonathan-gramain) [inline eve/main.yml]: I think you are missing one dash: should be `--gzip`

**Review by jonathan-gramain — APPROVED** [review]
looks good to me otherwise, approving!

**Review by miniscruff — APPROVED** [review]

**Review by lucieleonard — APPROVED** [review]

**Comment by lucieleonard** [comment]
Thank you!!!!!

File: eve/main.yml (hunk 1 above)
Comment (tcarmet) [reply eve/main.yml]: True, good catch, now I wonder how it worked :D

**Review by wabernat — APPROVED** [review]

**Pull request merged**